website_and_docs/content/documentation/grid/configuring_components/cli_options.en.md

```diff
@@ -215,6 +215,7 @@ pull request updating this page.
 | `--register-period` | int | `120` | How long, in seconds, will the Node try to register to the Distributor for the first time. After this period is completed, the Node will not attempt to register again. |
 | `--session-timeout` | int | `300` | Let X be the session-timeout in seconds. The Node will automatically kill a session that has not had any activity in the last X seconds. This will release the slot for other tests. |
 | `--vnc-env-var`| string | `START_XVFB` | Environment variable to check in order to determine if a vnc stream is available or not. |
+| `--hub` | string | `http://localhost:1234` | Configure event bus to the default ports. Setting `--grid-url`,`--publish-events` and `--subscribe-events` flags to their default values.|
```

Comment:
That is not entirely accurate, it does set it for the EventBus but also the Grid url. You can check the whole description [here](https://github.com/SeleniumHQ/selenium/blob/7213e08aa4b070547bced450f7b0998f49c322d5/java/src/org/openqa/selenium/grid/node/config/NodeFlags.java#L113-L123).